doc/source/data/working-with-tensors.rst

```diff
@@ -33,40 +33,51 @@ Batches of fixed-shape tensors
 
 If your tensors have a fixed shape, Ray Data represents batches as regular ndarrays.
 
-.. doctest::
```

Comment:
You can't copy the REPL prompts. They're not selectable.

<img width="760" alt="image" src="https://github.com/ray-project/ray/assets/26107013/9a4d8b97-9042-4948-993d-d269a6193e1c">

https://github.com/ray-project/ray/blob/790a16d4f5bc748cb405cb00c83b901118dda284/doc/source/conf.py#L118-L124